src/e2e/specs/dashboard.spec.ts

```diff
@@ -328,68 +327,50 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Footer`, () =>
 
     const clickOnATagCheckDomain = async (
       aTag: Locator,
-      expectedStrings: string[],
-    ): Promise<boolean> => {
+      host: string,
+      pathParts: string[],
+    ) => {
       const href = await aTag.getAttribute("href");
       if (href === null) return false;
 
-      const target = await aTag.getAttribute("target");
-
-      if (target === "_blank") {
-        //opens link in a new tab
-        const newPage = await context.newPage();
-        await newPage.goto(href);
-        const currentUrl = newPage.url();
-        await newPage.close();
-        return expectedStrings.every((expectedString) =>
-          currentUrl.includes(expectedString),
-        );
-      } else {
-        //opens link in the same tab
-        await page.goto(href);
-        const currentUrl = page.url();
-        await page.goBack();
-        return expectedStrings.every((expectedString) =>
-          currentUrl.includes(expectedString),
-        );
-      }
+      await page.goto(href);
+      const currentUrl = new URL(page.url());
+      const perceivedHost = currentUrl.hostname;
+      const perceivedPath = currentUrl.pathname;
+      expect(perceivedHost).toBe(host);
+      expect(pathParts.every((part) => perceivedPath.includes(part)));
```

Comment:
ah, interesting! I dont think this does anything since there aren't any matchers.
For example, this seems to pass locally for me in Playwright:

```js
expect(false);
```

So I _think_ this will pass regardless of whether the `.every()` includes all the substrings &mdash; unless we add `.toBe(true)` or `.toBeTruthy()`.